index.html

```diff
@@ -6,6 +6,9 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <script src="https://kit.fontawesome.com/f8112dc1f2.js" crossorigin="anonymous"></script>
   <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;900&display=swap" rel="stylesheet">
   <title>My Portfolio</title>
 </head>
 <body>
```

Comment:
- [ ] <b>REQUIRED:</b> On screen `> 376PX` there is a gap on the right side. that is because, In CSS, you set the width of the body to `23.5em` which is equal to `376px`, that actual not correct because not all mobile phone has the exact width of `376px`. the maximum width size of a mobile screen is `480px`. but that is beside the point. in mobile-first approach, everything must look the same until you adjust your app for other screen sizes(tablet or desktop)
![scrnli_9_16_2021_9-54-49 PM](https://user-images.githubusercontent.com/45217364/133677412-7e48fbc2-1415-4644-954e-0aa995559234.png)

Reply:
Hello  Rukundoeric.

Thanks for the review. I'm not sure I understand what I'm suppose to do. Please, I'd like you to explain further or if you don't mind we could have a meeting so you can explain to me better. Thanks  Rukundoeric

Reply:
Hello @audukehinde, here is the solution.
- For the body, don't force it to have a fixed size. just let it have the size of the available screen.
- And for the items, make sure its container has `display: flex and flex-wrap: wrap`

Reply:
Thanks so much. I'll see to that.

Reply:
Hey, thanks for your patience with me, I really appreciate it.

So, I made the changes you pointed out but I'm not sure I got it right and I'd love to get it right before asking for the second review. 

Please check my pull request link below to see if I've made the changes required.
https://github.com/audukehinde/portfolio/pull/2

Please, let me know if I did not get it right. 
I'm so sorry to disturb and thanks for your help.